### PR TITLE
Add the task initialization

### PR DESCRIPTION
**Pull request by luctus**

### CODE REVIEW

**Comment by sebastian-palma** [comment]
I'd also modify the lines with:

 1. /config/liam.yaml
 2. /lib/task/liam.rake

To:

 1. config/liam.yaml
 2. lib/task/liam.rake

As in the first case, they seem to make reference to a config and lib folders in the root directory. Would you mind updating them @luctus? 🥺

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Then execute this little rake tast that will generate some configuration files n
  1. /config/liam.yaml
```
Comment (luctus) [inline README.md]:
```suggestion
 1. config/liam.yaml
```

File: README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Then execute this little rake tast that will generate some configuration files n
...
  2. /lib/task/liam.rake
```
Comment (luctus) [inline README.md]:
```suggestion
 2. lib/task/liam.rake
```

**Comment by luctus** [comment]
done!

**Pull request merged**